deploy/complete/helm-chart/setup/values.yaml

```diff
@@ -76,4 +76,47 @@ grafana:
     - grafana-kubernetes-app
     - grafana-worldmap-panel
     - grafana-piechart-panel
-    - btplc-status-dot-panel
\ No newline at end of file
+    - btplc-status-dot-panel
+
+# https://github.com/helm/charts/blob/master/stable/jenkins/README.md
+jenkins:
```

Comment:
How much of these are defaults? Is it possible to simplify our values by deferring to the jenkins chart defaults?

Reply:
Quite a few - good catch.
eliminated most of it. The one thing I'd like to keep as a redundant one is the `serviceType=ClusterIP`. This is default, but that could change, since the default for the rest of the config is `NodePort`.

Reply:
@mvandervliet - having some issues running `hugo`. Will add docs as soon as that is resolved.

```
Building sites … ERROR 2020/06/23 23:57:06 render of "page" failed: execute of template failed: template: _default/single.html:1:3: executing "_default/single.html" at <partial "_open.html" .>: error calling partial: execute of template failed: template: partials/_open.html:5:7: executing "partials/_open.html" at <partial "menu.html" .>: error calling partial: "/Users/JGJOSEPH/dev/mushop/src/docs/themes/redwood/layouts/partials/menu.html:93:24": execute of template failed: template: partials/menu.html:93:24: executing "tree-nav" at <(not .Params.hidden) or $.showhidden>: can't give argument to non-function not .Params.hidden
```

Reply:
ok... this probably relates to #163, I'll take a look

Reply:
@mvandervliet - thanks for the help with `hugo`. Docs updated - the "how to setup a pipeline" section is scant right now. 
Will expand later as a HOL